docs/install-olm.md

```diff
@@ -0,0 +1,64 @@
+# How do I install OLM?
+
+The OLM is installable on Kubernetes clusters. It is already installed and available by default on 4.x OpenShift 
+clusters. For the following instructions to work, you must have a Kubernetes cluster running and the `kubectl` 
+command-line tool is able to find the information it needs to communicate with the API server of that cluster. For more 
```

Comment:
The comment about the configuring `kubectl` to speak with a cluster seems a bit verbose.